jobs/service-fabrik-broker/templates/bin/pre-start.erb

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -u # report the usage of uninitialized variables
+
+set +e
```

Comment:
why is set +e for the next three commands alone ?

Reply:
'sudo service keepalived stop' would be done irrespective of service is present or not. If in a vm the service is not present then, says unrecognised service and throws non zero.